src/routes/articles.js

```diff
@@ -216,7 +219,7 @@ router
 	.route('/:id')
 	.get(getArticle)
 	.delete(deleteArticle)
-	.patch(updateArticle)
+	.patch(validateRequest, updateArticle)
```

Comment:
so this will now be

```suggestion
	.patch(validateRequest(true, updatePostSchema), updateArticle)
```

you will have to do same for places already using the `validateSchema`. Import the needed schemas and pass it this way